syntax = "proto3";

message HelloRequest {
    string name = 1;
}

message HelloReply {
    string message = 1;
}

message HighOverheadRequest {
    repeated float req = 1;
};

message HighOverheadResponse {
    repeated float res = 1;
};

service Greeter {
    rpc SayHello(HelloRequest) returns (HelloReply) {}
    rpc SayGoodBye(HighOverheadRequest) returns (HighOverheadResponse) {}
}
